playground/ssr-react/__tests__/ssr-react.spec.ts

```diff
@@ -70,7 +71,8 @@ test('client navigation', async () => {
 })
 
 test(`circular dependencies modules doesn't throw`, async () => {
-  await page.goto(url)
+  await untilBrowserLogAfter(() => page.goto(url), 'hydrated')
```

Comment:
This is not necessary but I think it's better to call this to make all tests consistent.